Dockerfile.worker

```diff
@@ -151,6 +157,11 @@ RUN sed -i '1ilocal   all             all                                     tr
 
 RUN chmod a+rx $CODE
 
+WORKDIR /var/data/garmin/additional_data/
+# Fetch required additional data for garmin
+RUN wget -O /var/data/garmin/additional_data/bounds.zip http://osm2.pleiades.uni-wuppertal.de/bounds/latest/bounds.zip
+RUN wget -O /var/data/garmin/additional_data/sea.zip http://osm2.pleiades.uni-wuppertal.de/sea/latest/sea.zip
```

Comment:
Would a comment be appropriate that these URLs were found on http://www.mkgmap.org.uk/download/mkgmap.html ? I guess the URLs ever change, that'd be where we'd have a high chance of finding the new ones.